infrastructure/app/chart/energy-apps/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include"energy-apps.fullname" . }}
```

Comment:
Missing space after `include`. I don't actually know if that's a Helm syntax error, or if it will work anyway.

Reply:
Thanks! I've corrected that 👍🏻